include/crypto/x509.h

```diff
@@ -322,6 +322,10 @@ int ossl_x509_add_cert_new(STACK_OF(X509) **sk, X509 *cert, int flags);
 int ossl_x509_add_certs_new(STACK_OF(X509) **p_sk, STACK_OF(X509) *certs,
                             int flags);
 
+int X509_STORE_CTX_get_by_subject_int(const X509_STORE_CTX *vs,
```

Comment:
@DDvO I have some PR's that are fixing up internal symbols to be 
(1) prepended with ossl_
(2) use lower case if they are internal..

So this name for example could be
ossl_x509_store_ctx_get_by_subject().

Reply:
Thanks for this remark.
Changed the name to the one you proposed.
The declaration of the function in include/crypto/x509.h was not actually needed, so removed that.

Reply:
Yeah, right, in this case I could even have used a shorter name.